### PR TITLE
upgrade XRP to XRPL

### PR DESCRIPTION
**Pull request by nitsujlangston**

### CODE REVIEW

File: tests/xrp.js [hunk 1]
```diff
@@ -32,46 +33,91 @@ describe('XRP Tests', function() {
...
+    try {
+      txid = await rpcs.unlockAndSendToAddress({ currency, address: config.currencyConfig.sendTo, amount: '10000', secret: 'snoPBrXtMeMyMHUVTgbuqAfg1SUTb' });
+    } catch (err) {
+      expect(err).to.not.exist();
+    }
+
     expect(txid).to.have.lengthOf(64);
     assert(txid);
+    await xrpRPC.asyncRequest('ledger_accept');
+    // await xrpRPC.asyncRequest('ledger_accept');
```
Comment (escottalexander) [inline tests/xrp.js:118]: This comment can be removed.

**Review by escottalexander — DISMISSED** [review]
LGTM. I like all the added try/catchs and the additional tests. Everything looks a lot cleaner than before as well.

File: tests/xrp.js (hunk 1 above)
Comment (nitsujlangston) [reply tests/xrp.js]: thanks, fixed and pushed

**Review by escottalexander — APPROVED** [review]
This looks good. Re-ran the tests in docker and confirmed that everything works with the new version.

**Comment by kajoseph** [comment]
@nitsujlangston Why did you add the try catches in the tests? A throw will fail the test anyway and usually with a more helpful error than `expected ... to not exist`

**Review by kajoseph — CHANGES_REQUESTED** [review]
Just have one thing to fix. I'm still trying to understand something with the amounts. According to the docs, I should be able to create a new account with only 10+ XRP, but I tried with 100 XRP and it didn't work. It worked with 1000 XRP and I don't know why.

tl;dr - I'm still trying to make sure the `amount` handling is correct in sendTransaction

**Review by kajoseph — APPROVED** [review]
tACK
